tests/integration/integration_suite_test.go

```diff
@@ -249,14 +250,19 @@ func loadRowsAndCheckCount(t *testing.T, namespace string, rowsToLoad, rowsExpec
 
 func createMedusaSecretAndInstallDeps(t *testing.T, namespace, backend string) {
 	log.Println(Info("Creating medusa secret to access the backend"))
-	if backend == "local" {
-		return
-	} else if backend == "Minio" {
+	switch backend {
+	case "Minio":
 		DeployMinioAndCreateBucket(t, "k8ssandra-medusa")
 		CreateMedusaSecretWithFile(t, namespace, "secret/medusa_minio_secret.yaml")
-	} else {
-		// S3
-		CreateMedusaSecretWithFile(t, namespace, "~/medusa_secret.yaml")
+	case "s3":
```

Comment:
This change was intended to allow multiple backends to be tested during the integration tests, but I realize that the GH workflow definition is broken. I will look into a fix today.

Reply:
Fixed now.